src/DashboardService.Models.Dto/Responses/BoardResponse.cs

```diff
@@ -12,5 +12,5 @@ public class BoardResponse
 
   public bool IsActive { get; set; }
 
-  /* public GroupInfo Group {get; set;} */
-}
+  /* public IEnumerable<GroupInfo> Groups {get; set;} */
```

Comment:
@SolovyovDmitry may be add include project and remove projectId?

Reply:
Don't think that adding parent entities is good idea